kbase-extension/scss/partials/_bootstrapHelper.scss

```diff
@@ -98,6 +98,10 @@
   }
 }
 
+.btn {
```

Comment:
moved this out of the `narrative.scss` partial since it is to alter the bootstrap css.